app/services/deployment/configuration.rb

```diff
@@ -9,10 +9,12 @@ class Configuration
     attribute :deployment_configuration_id, Integer
     attribute :application_url, String
     attribute :addons, Array, of: String
+    attribute :web_processes, Hash
```

Comment:
I suggest to use type instead of Hash:
```ruby
class WebProcess
  attribute :name
  attribute :command

  # will be implemented later
  # attribute :number
end
```

Reply:
np 👍